hibernate-search-plugins/org.jboss.tools.hibernate.search/src/org/jboss/tools/hibernate/search/HSearchConsoleConfigurationPreferences.java

```diff
@@ -0,0 +1,39 @@
+package org.jboss.tools.hibernate.search;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.hibernate.console.ConsoleConfiguration;
+import org.hibernate.console.KnownConfigurations;
+
+public class HSearchConsoleConfigurationPreferences {
+	
+	private static final String DEFAULT_HIBERNATE_SEARCH_VERSION = "4.3";
```

Comment:
Ouch :( Could we please default to the most recent stable & supported version? That's `5.5.5.Final`